mlrun/model.py

```diff
@@ -1581,10 +1582,13 @@ def get_templated_path(self):
         return self.full_path_template
 
     def get_absolute_path(self):
-        if self.run_id:
-            return self.full_path_template.format(run_id=self.run_id)
-        else:
-            return self.full_path_template
+        project = mlrun.get_current_project(silent=True)
```

Comment:
Well... I'm not 100% sure this will always work. It really depends whether the user has a project context set (by using `load_project`, `get_or_create_project` etc.). It's not always the case in Jupyter, and doing local ingest may fail in that case. What's worse - if you ingest a fset from project A while your context holds project B, you will get the wrong project here. The fset's project must take precedence over the current user context.
I suggest that you add an optional `project=None` parameter to this function, and verify you pass the project whenever calling this function and it makes sense. Specifically, in `feature_set.get_target_path()` you need to pass the fset's project there when it's calling it (`self.metadata.project`). You can still look for `get_current_project` if you don't get this parameter, but first use that.
Need to also look at other cases where this is called (I don't think there are many) and make sure the proper project is passed in any of them.